### PR TITLE
WIP: What a messy PR!

### PR DESCRIPTION
**Pull request by ericdill**
This PR is mostly a total overhaul of replay's guts.  Still working on it, but it's getting there.
## Must Haves for this PR
- [x] Successfully throw a dataframe at the scalar model and view the results
- [x] Plot each column of a dataframe against its index
- [x] Plot columns of a dataframe against each other
- [x] Add tests for the Atom Models backing the enaml code
# +4% coverage!
## Things this PR is not going to do
- Deal with plotting error because matplotlib's `axes.errorbar` is a sufficiently different beast than `axes.plot` that I need @tacaswell around so we can figure out how to deal with this... See #97 
## Running update

656e0b0: plotting against time works with the new `ColumnModel`, but you can no longer turn on/off data
98bd14e: apparently it was pretty easy to fix the problem with turning on/off data...
16ad3a2: All flavors of scalar plotting now work.
28f373c: Let the testing olympics begin...

### CODE REVIEW

File: dataportal/replay/scalar/model.py [hunk 1]
```diff
@@ -142,6 +151,33 @@ def state(self):
...
+    _column_address = Typed(object) # really this is a tuple or a string
+    dataframe = Typed(pd.DataFrame)
+
+    @property
+    def data(self):
+        return np.asarray(self.dataframe[self._column_address].values)
+
+    @property
+    def index(self):
+        return np.asarray(self.dataframe[self._column_address].index)
```
Comment (danielballan) [inline dataportal/replay/scalar/model.py]: `self.dataframe['time'].values`

**Comment by ericdill** [comment]
yay! seems that my replay tests work on travis... the only tests that are failing are the ones related to the mds `find_event_by_runstart_id` thing that isn't on master

**Comment by ericdill** [comment]
@tacaswell @danielballan This can be deployed tomorrow.

**Comment by danielballan** [comment]
@tacaswell Did this get in in time to be tagged? If not, my vote is re-tag.
We need replay today.

**Comment by tacaswell** [comment]
No, I am planning to tag this first thing at the office.

**Comment by danielballan** [comment]
Changes to broker and muxer look fine. Good to merge.

**Pull request merged**